src/GitVersionCore/Core/Abstractions/Git/ITag.cs

```diff
@@ -0,0 +1,9 @@
+namespace GitVersion
+{
+    public interface ITag
```

Comment:
```suggestion
    public interface ITag : IEquatable<ITag>, IComparable<ITag>
```

Reply:
Agree, I will change that in a subsequent PR